lib/main.js

```diff
@@ -7,14 +7,14 @@
 
     var _ = require('lodash');
 
-    var Main = function(apiKey) {
+    var Main = function(apiUrl, apiKey) {
         var _services = ['name', 'search', 'thumbnail']
             , Inherits = require('./inherits')
             , self = this;
 
         // Make some configuration;
         this.options = {
-            api_url: "http://api.pipl.com/search/v4/",
+            api_url: apiUrl,
```

Comment:
I would prefer it to be:

```javascript
api_url: apiUrl || "http://api.pipl.com/search/v4/";
```

Because we cannot change the signature of the constructor without a fallback. So, if someone explicitly passes, OK, if not, use v4.